decode/clarity.ts

```diff
@@ -46,83 +56,109 @@ export function decode(input: string | IPayload, augmentations: IAugmentation =
             case Event.TouchCancel:
             case Event.TouchEnd:
             case Event.TouchMove:
-                event = interaction(entry);
-                payload.analytics.push(event);
+                if (payload.pointer === undefined) { payload.pointer = []; }
+                payload.pointer.push(interaction.decode(entry) as Decode.PointerEvent);
+                break;
+            case Event.Scroll:
+                if (payload.scroll === undefined) { payload.scroll = []; }
+                payload.scroll.push(interaction.decode(entry) as Decode.ScrollEvent);
+                break;
+            case Event.Resize:
+                if (payload.resize === undefined) { payload.resize = []; }
+                payload.resize.push(interaction.decode(entry) as Decode.ResizeEvent);
+                break;
+            case Event.Selection:
+                if (payload.selection === undefined) { payload.selection = []; }
+                payload.selection.push(interaction.decode(entry) as Decode.SelectionEvent);
+                break;
+            case Event.Change:
```

Comment:
Very vague event name. Can you rename to something clearer (even if this is what the DOM event is called). For example, InputChanged?

Reply:
Done => InputChange